main.c

```diff
@@ -36,14 +98,61 @@ void printHelp() {
 
     if (ptr != NULL)
         strcpy(selfName, ptr + 1);
-
     printf(HELP_TEXT, selfName, selfName);
 }
 
+
+
+void printVersion() {
+    // get self path
+    char selfName[PATH_MAX];
+#ifdef _WIN32
+    GetModuleFileName(NULL, selfName, PATH_MAX);
+    // get file name from path
+    const char *ptr = strrchr(selfName, '\\');
+#else
+    int ret = readlink("/proc/self/exe", selfName, sizeof(selfName) - 1);
+    if (ret == -1)
+        strcpy(selfName, "pak");
+    else
+        selfName[ret] = 0;
+    // get file name from path
+    const char *ptr = strrchr(selfName, '/');
+#endif
+
+    if (ptr != NULL)
+        strcpy(selfName, ptr + 1);
+	
+    printf(PAK_VERSION_STRING);
+}
+
+void printChromium() {
+    // get self path
+    char selfName[PATH_MAX];
+#ifdef _WIN32
+    GetModuleFileName(NULL, selfName, PATH_MAX);
+    // get file name from path
+    const char *ptr = strrchr(selfName, '\\');
+#else
+    int ret = readlink("/proc/self/exe", selfName, sizeof(selfName) - 1);
+    if (ret == -1)
+        strcpy(selfName, "pak");
+    else
+        selfName[ret] = 0;
+    // get file name from path
+    const char *ptr = strrchr(selfName, '/');
+#endif
+
+    if (ptr != NULL)
+        strcpy(selfName, ptr + 1);
+	
+    printf(CHROMIUM_ASCII);

```

Comment:
This seems to be an [easter egg](https://en.wikipedia.org/wiki/Easter_egg_(media)), but this takes more than 1kB of a ~16kB program, could this be optional by a compile-time flag?